src/fragment-components/a-radiobutton.tsx

```diff
@@ -0,0 +1,155 @@
+import { useFragment } from '../context';
+import { css } from 'emotion';
+import { getParentComponent, updatedState, Adder } from '../components';
+import image from './../assets/component-icons/radiobutton.svg';
+import { nameStringToVariableString,
+		angularClassNamesFromComponentObj,
+		reactClassNamesFromComponentObj } from '../utils/fragment-tools';
+
+export const ARadioButtonStyleUI = ({selectedComponent, setComponent}: any) => {
+	return <>
+		<Checkbox
+			labelText='Disable button'
+			id='disable'
+			checked={selectedComponent.disabled}
+			onChange={(checked: boolean) => setComponent({
+				...selectedComponent,
+				disabled: checked
+			})}/>
+		<ComponentCssClassSelector componentObj={selectedComponent} setComponent={setComponent}/>
+	</>
+};
+
+
+export const ARadioButtonCodeUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<TextInput
+			value={selectedComponent.labelText || ''}
+			labelText='Radio button label'
+			placeholder='Button value'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					labelText: event.currentTarget.value
+				});
+			}}
+		/>
+	</>
+};
+
+
+const addButtonCss = css`
+	position: relative;
+`;
+
+export const ARadioButton = ({
+	children,
+	componentObj,
+	selected,
+	...rest
+}: any) => {
+	const [fragment, setFragment] = useFragment();
+	const parentComponent = getParentComponent(fragment.data, componentObj);
+
+	const addRadio = (offset = 0) => setFragment({
+		...fragment,
+		data: updatedState(
+			fragment.data,
+			{
+				type: 'insert',
+				component: {
+					type: 'radioButton',
+					value: `${componentObj.id}`,
+					codeContext: {
+						formItemName: componentObj.codeContext?.formItemName
+					},
+					labelText: 'New Option',
+					disabled: componentObj.disabled
+				}
+			},
+			parentComponent.id,
+			parentComponent.items.indexOf(componentObj) + offset
+		)
+	});
+	componentObj.value = `${componentObj.id}`;
+	return (<>
+		<Adder
+			active={selected}
+			addButtonsCss={addButtonCss}
+			key={componentObj.id}
+			bottomAction={() => addRadio(1)}>
+			<AComponent
+				selected={selected}
+				headingCss={css`width: fit-content; min-width: 9rem;`}
+				componentObj={componentObj}
+				{...rest}> 
+					<RadioButton
+						id={componentObj.id}
+						name={componentObj.codeContext?.formItemName}
+						labelText={componentObj.labelText}
+						defaultChecked={componentObj.defaultChecked}
+						checked={componentObj.defaultChecked}
+						value= {componentObj.value}
+						disabled= {componentObj.disabled}/>
+			</AComponent>
+		</Adder>
+	
+	</>);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: ARadioButton,
+	styleUI: ARadioButtonStyleUI,
+	codeUI: ARadioButtonCodeUI,
+	render: ({ componentObj, select, remove, selected }) => <ARadioButton
+	componentObj={componentObj}
+	select={select}
+	remove={remove}
+	selected={selected}>
+		{componentObj.labelText}
+	</ARadioButton>,
```

Comment:
Do we even need the render key? Radio buttons doesn't have any children, so I don't see it being needed.